### DISCUSSION TITLE
Tell me about the GPU you are using(あなたの使っているGPUについて教えてください).

### DISCUSSION THREAD

**Discussion by suika10000** [NAIST-SE/AttractingContributors#487] [General]
Please tell me about the GPU you are using. 
あなたの使っているGPUについて教えてください．

For research or personal use, either OK.
研究用でも，個人用でもどちらでもかまいません．

Please follow this link to join us.
こちらのリンクから、参加してください。

My repo: https://github.com/suika10000/AttractingContributors-GPU

**Comment by ploychanok** [comment]
I created a pull request ‘add m1’.

Would you mind contribute mine too? 
https://github.com/ploychanok/AttractingContributors-WithinThreeWords

thank you

**Reply by suika10000** [reply]
Thank you.
Here I go.
